client/src/utils/api.js

```diff
@@ -1,22 +1,16 @@
-const axios = require('axios');
+const axios = require("axios");
 
 const instance = axios.create({
-  baseURL: 'http://localhost:5000',
+  baseURL: "http://localhost:5000",
```

Comment:
Let's change this to https://collective-liberty.vercel.app/

Reply:
It has been done!!